docs/sources/faq.md

```diff
@@ -234,46 +217,48 @@ When building Docker images on Debian and Ubuntu you may have seen errors like:
     unable to initialize frontend: Dialog
 
 These errors don't stop the image from being built but inform you that the
-installation process tried to open a dialog box, but was unable to. 
-Generally, these errors are safe to ignore.
+installation process tried to open a dialog box, but was unable to. Generally,
+these errors are safe to ignore.
 
-Some people circumvent these errors by changing the `DEBIAN_FRONTEND` 
+Some people circumvent these errors by changing the `DEBIAN_FRONTEND`
 environment variable inside the Dockerfile using:
 
     ENV DEBIAN_FRONTEND=noninteractive
 
-This prevents the installer from opening dialog boxes during installation 
-which stops the errors.
+This prevents the installer from opening dialog boxes during installation which
+stops the errors.
 
-While this may sound like a good idea, it *may* have side effects. 
-The `DEBIAN_FRONTEND` environment variable will be inherited by all 
-images and containers built from your image, effectively changing
-their behavior. People using those images will run into problems when
-installing software interactively, because installers will not show
-any dialog boxes.
+While this may sound like a good idea, it *may* have side effects. The
+`DEBIAN_FRONTEND` environment variable will be inherited by all images and
+containers built from your image, effectively changing their behavior. People
+using those images will run into problems when installing software
+interactively, because installers will not show any dialog boxes.
 
 Because of this, and because setting `DEBIAN_FRONTEND` to `noninteractive` is
 mainly a 'cosmetic' change, we *discourage* changing it.
 
-If you *really* need to change its setting, make sure to change it
-back to its [default value](https://www.debian.org/releases/stable/i386/ch05s03.html.en) 
+If you *really* need to change its setting, make sure to change it back to its
+[default value](https://www.debian.org/releases/stable/i386/ch05s03.html.en)
 afterwards.
 
```

Comment:
@thaJeztah Obviously, Googling stroopwafels is insufficient. I must travel to the Netherlands and experience them to truly know the term.

I thought you meant my lines but I checked the wrap and they were good. I checked some others and they were not so thought maybe my editor wonked them...my fault entirely for 1) not confirming and 2) understanding the general rule. Alas, I got no stroopwafels --- the American equivalent is perhaps the pancake.